client/src/components/ManageBookings/BookModule/CurrentBookings.tsx

```diff
@@ -0,0 +1,56 @@
+import { Grid, Paper, Typography, Box, CssBaseline, CircularProgress } from '@material-ui/core';
+import EachBooking from './EachBooking/EachBooking';
+import { Request } from '../../../interface/Request';
+
+import useStyles from './useStyles';
+
+interface Props {
+  bookings: Array<Request>;
+  handleAccept: (arg1: boolean, arg2: number) => void;
+}
+
+const CurrentBookings = ({ bookings, handleAccept }: Props): JSX.Element => {
+  const classes = useStyles();
+  const now = new Date();
+
+  let pastBookings = [...bookings];
+  pastBookings = pastBookings.filter((request) => request.date < now);
+  let futureBookings = [...bookings];
+  futureBookings = futureBookings.filter((request) => request.date >= now);
+
+  if (!bookings) return <CircularProgress />;
+  return (
+    <Grid>
+      <CssBaseline />
+      <Paper elevation={3}>
+        <Grid className={classes.root}>
+          <Typography className={classes.title}>Current bookings:</Typography>
+          {futureBookings.map(
+            (request, i) => (
+              i++,
+              (
+                <EachBooking
+                  key={i}
```

Comment:
Indexes as keys on React is not the best practice however, not critical. If anything on request is constant and can be used that would be better. If not, don't worry about it.

Reply:
Good to know! I'll change this out for the mongoID with the integration then.